posts/2022-09-01-java-19-22.0.0.10-beta.adoc

```diff
@@ -0,0 +1,291 @@
+---
+layout: post
+title: "Java 19 and MicroProfile OpenTelemetry previews in Open Liberty 22.0.0.10-beta"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/reecenana
+author_github: https://github.com/reecenana
+seo-title: Java 19 and MicroProfile OpenTelemetry previews in Open Liberty 22.0.0.10-beta - OpenLiberty.io
+seo-description: Java SE 19 and MicroProfile OpenTelemetry 1.0 are released this month so we bring you previews of both in this Open Liberty 22.0.0.10-beta. MicroProfile OpenTelemetry provides a standards-based set of APIs, SDKs, tooling and integrations that are designed for the creation and management of telemetry data such as traces, metrics, and logs. 
+blog_description: Java SE 19 and MicroProfile OpenTelemetry 1.0 are released this month so we bring you previews of both in this Open Liberty 22.0.0.10-beta. MicroProfile OpenTelemetry provides a standards-based set of APIs, SDKs, tooling and integrations that are designed for the creation and management of telemetry data such as traces, metrics, and logs. 
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= Java 19 and MicroProfile OpenTelemetry previews in Open Liberty 22.0.0.10-beta
+Reece Nana <https://github.com/reecenana>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+Java SE 19 and MicroProfile OpenTelemetry 1.0 are released this month so we bring you previews of both in this Open Liberty 22.0.0.10-beta. Java 19 provides several new and changed features, including record patterns and virtual threads. MicroProfile OpenTelemetry provides a standards-based set of APIs, SDKs, tooling and integrations that are designed for the creation and management of telemetry data such as traces, metrics, and logs.

```

Comment:
It is used both ways, but I believe it should be MicroProfile Telemetry 1.0.  MicroProfile exposes the Open Telemetry function, but the MP name does not include the word Open.

Reply:
To be clear, I am looking for anywhere MicroProfile OpenTelemetry be updated to MicroProfile Telemetry.  If the word MicroProfile is not before OpenTelemetry that can remain.

Reply:
Bah! Sorry, that's my fault. I'll fix.

Reply:
Fixed